CHANGELOG.md

```diff
@@ -35,6 +35,22 @@ release.
 
 ## [Unreleased]
 
+### Changed
+- isisVarInit.py no longer writes a "cat" statement by default to the activate scripts which cause the ISIS version information to be written on conda activate.  This can be included in those scripts via a command line option to isisVarInit.py.  Also, a quiet option is provided to isisVarInit.py to suppress its own writing to standard out, if needed.
```

Comment:
This doesn't appear to be in 5.0.1, it should probably not be added to the changelog for this branch.